lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/cw/XwikiController.java

```diff
@@ -147,6 +148,14 @@ public InputStream getModel(String modelSetId, ModelSetType type) throws LpRestE
     public String startSimulation(String modelId, String currentUser, Collection<UserData> potentialUsers)
         throws LpRestException
     {
+    	currentUser = this.removePrefixes(currentUser);
```

Comment:
This can be done directly on `CWXwikiBridge`, CP shouldn't have to do that because it shouldn't have knowledge about these weird user ID.